### PR TITLE
fix(a380x): Baro Knob Fix

### LINKED ISSUE #9239: STD Baro knob
### Aircraft Version

Development

### Build info

```json
v0.12.0
```

### Describe the bug

When setting standard I have to pull the baro knob

### Expected behavior

On the 380/350 aircraft irl it is push for STD

### Steps to reproduce

try setting STD

### Additional info (optional)

_No response_

### Discord Username (optional)

l23

### PR DESCRIPTION
**Pull request by danestfanous**
Changed the baro knob to set to STD mode when pushing the knob, and BARO mode when pulling the knob. 

Fixes #9239 

## Summary of Changes

-Changed the code for the push and pull functionality of the knob.
-Changed the tooltip content to reflect the new actions on the knob. 

## References

https://youtube.com/clip/Ugkxipw1VOv31d31ta8XJR8IJ5DnCf4xMfjU?si=q8goGPl1rLZjNfzy

## Additional context

Discord username (if different from GitHub):
shadowsurfer0815

## Testing instructions

Load into the FBW A380X and start up the aircraft. Push the baro knob (up arrow) to set it to STD. Pull the baro knob (down arrow) to set QNH. 

## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo** or **flybywire-aircraft-a380-842** download link at the bottom of the page

### CODE REVIEW

**Review by 2hwk — APPROVED** [review]

File: fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/legacy/generated/A32NX_Interior_FCU.xml [hunk 1]
```diff
@@ -10,9 +10,9 @@
             <ANIMTIP_0_ON_CURSOR>TurnRight</ANIMTIP_0_ON_CURSOR>
             <ANIMTIP_1>TT:COCKPIT.TOOLTIPS.AUTOPILOT_PANEL_BARO_KNOB_DECREASE</ANIMTIP_1>
             <ANIMTIP_1_ON_CURSOR>TurnLeft</ANIMTIP_1_ON_CURSOR>
-            <ANIMTIP_2>TT:COCKPIT.TOOLTIPS.BARO_REF_SET_STD</ANIMTIP_2>
+            <ANIMTIP_2>TT:COCKPIT.TOOLTIPS.BARO_ENABLE_SELECTION</ANIMTIP_2>
             <ANIMTIP_2_ON_CURSOR>DownArrow</ANIMTIP_2_ON_CURSOR>
-            <ANIMTIP_3>TT:COCKPIT.TOOLTIPS.BARO_ENABLE_SELECTION</ANIMTIP_3>
+            <ANIMTIP_3>TT:COCKPIT.TOOLTIPS.BARO_REF_SET_STD</ANIMTIP_3>
```
Comment (tracernz) [inline fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/legacy/generated/A32NX_Interior_FCU.xml:15]: Do these keys actually exist in fs-base?

File: fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/legacy/generated/A32NX_Interior_Misc.xml [hunk 2]
```diff
@@ -608,10 +608,10 @@
                                 <CLOCKWISE_CONDITION>(L:XMLVAR_Baro#BARO_ID#_Mode) #MODE_STD# !=</CLOCKWISE_CONDITION>
                                 <ANTICLOCKWISE_CONDITION>(L:XMLVAR_Baro#BARO_ID#_Mode) #MODE_STD# !=</ANTICLOCKWISE_CONDITION>
                                 <PULL_CODE>
-                                    #MODE_STD# (&gt;L:XMLVAR_Baro#BARO_ID#_Mode)
+                                    (L:XMLVAR_Baro#BARO_ID#_MODE) #MODE_STD# == if{ #MODE_BARO# (&gt;L:XMLVAR_Baro#BARO_ID#_Mode) } els{ (L:XMLVAR_Baro#BARO_ID#_Mode) ! (&gt;L:XMLVAR_Baro#BARO_ID#_Mode) }
                                 </PULL_CODE>
                                 <PUSH_CODE>
-                                    (L:XMLVAR_Baro#BARO_ID#_Mode) #MODE_STD# == if{ #MODE_BARO# (&gt;L:XMLVAR_Baro#BARO_ID#_Mode) } els{ (L:XMLVAR_Baro#BARO_ID#_Mode) ! (&gt;L:XMLVAR_Baro#BARO_ID#_Mode) }
+                                    #MODE_STD# (&gt;L:XMLVAR_Baro#BARO_ID#_Mode)
```
Comment (tracernz) [inline fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/legacy/generated/A32NX_Interior_Misc.xml:614]: Is this even still used?

Comment (danestfanous) [reply fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/legacy/generated/A32NX_Interior_Misc.xml:614]: No that's not being used. I will remove it right now.

**Review by danestfanous — COMMENTED** [review]
Updated and removed the unused code. The baro knob still works as expected.

**Review by tracernz — APPROVED** [review]

**Comment by Awemeter** [comment]
Quality Assurance Trainee Report

Discord Username: Awemeter 
Object of testing : #9252
Aircraft: A380X 
Tier of Testing: 1
Date: 11/5/2024 

Testing Process: 
 
Downloaded the PR artifact for A380
Placed it in the community folder along side stable version 12.1.0
Loaded into aircraft and verified correct behavior (Push the baro knob (up arrow) to set it to STD. Pull the baro knob (down arrow) to set QNH.)
Verified tooltips are displayed correctly
Testing Results: 
Passed 

Negatives:       


Conclusions: 

Looks good!
Media:

**Pull request merged**